src/test/java/jerry/codecamp/DogTest.java

```diff
@@ -126,12 +126,64 @@ public void timePassesForDog_increasesTiredLevel(){
     }
 
     @Test
-    public void timePassesForDog_increasesThirstLevel(){
+    public void timePassesForDog_increasesThirstLevel() {
         dog.timePassesForDog();
         assertEquals(5, dog.getThirstLevel());
 
         dog.timePassesForDog();
         assertEquals(10, dog.getThirstLevel());
     }
 
+    @Test
+    public void giveTheDogASnack_increasesHungerLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(- 10, dog.getHungerLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getHungerLevel());
+
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesLonelyLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(-10, dog.getLonelyLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-20, dog.getLonelyLevel());
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesDirtyLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(5, dog.getDirtyLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(10, dog.getDirtyLevel());
+
+        // hungerLevel -= 10;
+        //        lonelyLevel -= 10;
+        //        dirtyLevel += 5;
+        //        thirstLevel += 5;
+        //        boredLevel -= 5;
+
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesThirstLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(5, dog.getThirstLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(10, dog.getThirstLevel());
+    }
+
+    @Test
+    public void giveTheDogASnack_increasesBoredLevel() {
+        dog.giveTheDogASnack();
+        assertEquals(-5, dog.getBoredLevel());
+
+        dog.giveTheDogASnack();
+        assertEquals(-10, dog.getBoredLevel());
```

Comment:
Create a setter OR do something that makes the boredom go higher before trying to go down.

Reply:
do you mean dont go below zero?

Reply:
Oh yes.  don't go below 0.